.ci.yaml

```diff
@@ -19,27 +19,41 @@ platform_properties:
           {"dependency": "curl", "version": "version:7.64.0"}
         ]
   linux_android:
+    dimensions:
+      kvm: "1"
```

Comment:
shall we create a new platform with `linux_android_em`? Similar to what we did for framework repo.

Reply:
Yeah I can do that.